joblib/test/test_parallel.py

```diff
@@ -940,3 +942,43 @@ def __reduce__(self):
     with warns(DeprecationWarning):
         with raises(ValueError):
             delayed(UnpicklableCallable(), check_pickle=True)
+
+
+@with_multiprocessing
+@parametrize('backend', ['multiprocessing', 'loky'])
+def test_backend_batch_statistics_reset(backend):
```

Comment:
Can you make sure this test failing on master?

The number I chose in my snippet were because the difference was really big. IIRC I had bigger differences in master when having 1000 of ~1ms tasks than 100 of ~10ms tasks.

Reply:
> The number I chose in my snippet were because the difference was really big. IIRC I had bigger differences in master when having 1000 of ~1ms tasks than 100 of ~10ms tasks.

The problem with that number is that it failed all the time on Appveyor and this doesn't seem to be related to the same thing.

Reply:
> Can you make sure this test failing on master?

Yes it does: just comment the line `self.reset_batch_size()` in `terminate()` fails for both backends.

Reply:
>  having 1000 of ~1ms tasks than 100 of ~10ms tasks

I changed that to 500 of ~2ms tasks and it passes on Appveyor.